automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -121,55 +121,77 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
 
     cluster.googleProject shouldBe expectedProject
     cluster.clusterName shouldBe expectedName
-    cluster.stagingBucket shouldBe 'defined
 
-    implicit val patienceConfig: PatienceConfig = storagePatience
-    googleStorageDAO.bucketExists(GcsBucketName(cluster.stagingBucket.get.value)).futureValue shouldBe true
     labelCheck(cluster.labels, expectedName, expectedProject, cluster.creator, clusterRequest)
+
+    if (bucketCheck) {
+      cluster.stagingBucket shouldBe 'defined
+
+      implicit val patienceConfig: PatienceConfig = storagePatience
+      googleStorageDAO.bucketExists(GcsBucketName(cluster.stagingBucket.get.value)).futureValue shouldBe true
+    }
+
     cluster
   }
 
-  def createCluster(googleProject: GoogleProject, clusterName: ClusterName, clusterRequest: ClusterRequest, monitor: Boolean)(implicit token: AuthToken): Cluster = {
+  def createCluster(googleProject: GoogleProject,
+                    clusterName: ClusterName,
+                    clusterRequest: ClusterRequest,
+                    monitor: Boolean,
+                    apiVersion: ApiVersion = V1)
+                   (implicit token: AuthToken): Cluster = {
     // Google doesn't seem to like simultaneous cluster creates.  Add 0-30 sec jitter
     Thread sleep Random.nextInt(30000)
 
-    val clusterTimeResult = time(Leonardo.cluster.create(googleProject, clusterName, clusterRequest))
-    //TODO Uncomment the following line when the response time is truly less than 10s
-    //clusterTimeResult.duration should be < tenSeconds
-    logger.info("Time to get cluster create response::" + clusterTimeResult.duration)
+    val clusterTimeResult = time(Leonardo.cluster.create(googleProject, clusterName, clusterRequest, apiVersion))
+    logger.info("Time to get cluster create response:" + clusterTimeResult.duration)
+    // TODO Uncomment out the following when the response time is indeed less than 10 seconds
+    // clusterTimeResult.duration should be < tenSeconds
```

Comment:
The `sleep` in `Leonardo.get()` causes the assertion above to fail even though the response time is sub-second. There may be a way to make that assertion such that the sleep doesn't cause it to fail.